src/App.css

```diff
@@ -28,3 +28,131 @@ html {
 	justify-content: center;
 	font-family: 'Share Tech Mono', monospace;
 }
+
+.dialog-container {
+	display: flex;
+	position: fixed;
+	top: 0;
+	left: 0;
+	bottom: 0;
+	right: 0;
+	z-index: 2;
+}
+
+.dialog-container[aria-hidden='true'] {
+	display: none;
+}
+
+.dialog-overlay {
+	position: fixed;
+	top: 0;
+	left: 0;
+	bottom: 0;
+	right: 0;
+	background-color: rgba(43, 46, 56, 0.9);
+	animation: fade-in 200ms both;
+}
+
+.dialog-content {
+	font-family: 'Share Tech Mono', monospace;
+	background-color: var(--yellow);
+	margin: auto;
+	z-index: 2;
+	position: relative;
+	animation: fade-in 400ms 200ms both, slide-up 400ms 200ms both;
+	padding: 1em;
+	max-width: 90%;
+	width: 600px;
+	border-radius: 2px;
+	color: var(--blue);
+}
+
+@media screen and (min-width: 700px) {
+	.dialog-content {
+		padding: 2em;
+	}
+}
+
+@keyframes fade-in {
+	from {
+		opacity: 0;
+	}
+}
+
+@keyframes slide-up {
+	from {
+		transform: translateY(10%);
+	}
+}
+
+.dialog-title {
+	margin: 0;
+	font-size: 1.25em;
+}
+
+.dialog-close {
+	position: absolute;
+	top: 0.5em;
+	right: 0.5em;
+	border: 0;
+	padding: 0;
+	background-color: transparent;
+	font-weight: bold;
+	font-size: 1.25em;
+	width: 1.2em;
+	height: 1.2em;
+	text-align: center;
+	cursor: pointer;
+	transition: 0.15s;
+}
+
+@media screen and (min-width: 700px) {
+	.dialog-close {
+		top: 1em;
+		right: 1em;
+	}
+}
+
+* {
```

Comment:
This box-sizing can be deleted, since box-sizing is inherited from the index.css file that was provided for us!